src/output.c

```diff
@@ -30,6 +30,14 @@
 #include "view.h"
 #include "xwayland.h"
 
+static unsigned int
+get_tearing_retry_count(struct output *output)
+{
+	/* Two seconds worth of frames, guessing 60Hz if refresh is zero */
+	unsigned int refresh = output->wlr_output->refresh;
+	return refresh ? refresh / 500 : 120;
```

Comment:
In the interest of Postel's law:
```suggestion
	return refresh > 0 ? refresh / 500 : 120;
```